src/ui/components/usersListTable/UserRowMenu.tsx

```diff
@@ -0,0 +1,91 @@
+import { FC } from "react";
+import { getImageUrl } from "../../../utils";
+import { UserWithRecipeId } from "../../pages/usersList/types";
+
+export type UserRowMenuProps = {
+	onSelect?: (user: UserWithRecipeId) => void;
+	onDelete?: (user: UserWithRecipeId) => void;
+	onChangeEmail?: (user: UserWithRecipeId) => void;
+	onChangePassword?: (user: UserWithRecipeId) => void;
```

Comment:
In what condition would you want these props to be undefined?

Reply:
Just in case there is another page that uses similar User Menu, so it can be flexible and show only the correct button based on the available callbacks. 

Also. currently in this phase, it only has the `onSelect` feature, so it will only display the select user button.

Reply:
props is now mandatory